CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes for each version of this project will be documented in this file.
 
+## 15.1.33
+
+### General
+- `IgxButtonGroup`:
+    - Reverted cancellable on `selected` and `deselected` events (added in 15.1.0) as it was breaking firing order.
```

Comment:
This can't be right, 15.1.0 was a while back and the change being reverted is fairly recent.
No idea why the merged commit doesn't list it's specific tags for me, but I do see it in the changes listed for [15.1.24](https://github.com/IgniteUI/igniteui-angular/releases/tag/15.1.24) which fits the time slot, so:
```suggestion
    - Reverted cancellable on `selected` and `deselected` events (added in 15.1.24) as it was breaking firing order and related handling.
```